2-webpack/src/index.js

```diff
@@ -29,7 +30,7 @@ const onCheckboxCheck = (todoItemTextElement, checkboxElement, todoItemContainer
     if (checkboxElement.checked){
```

Comment:
let's change this whole function..
1. items in todoListItems should have o(1) access (hint, don't use an arrray)
2. write simpler function that does the if/ else (no more than 2 lines